README.md

```
@@ -1,2 +1,187 @@
+|                                                                                            **최현호**<br><sup>[CHH01](https://github.com/CHH01)</sup>                                                                                            |                           **김성경**<br><sup>[conradrado](https://github.com/hyunjae3458)</sup>                            |                               **이다솔**<br><sup>[LeeDyol](https://github.com/LeeDyol)</sup>                               |                              **이승민**<br><sup>[chosi123](https://github.com/chosi123)</sup>                              |                              **황민재**<br><sup>[rorm0819](https://github.com/rorm0819)</sup>                              |                              **신지연**<br><sup>[Nooroong](https://github.com/Nooroong)</sup>                              |
+| <img src="https://img.shields.io/badge/Leader-%23F05032?style=for-the-badge&logo=git&logoColor=white" alt="Leader" /> <img src="https://img.shields.io/badge/Backend-007ACC?style=for-the-badge&logo=spring&logoColor=white" alt="Backend" /> | <img src="https://img.shields.io/badge/Backend-007ACC?style=for-the-badge&logo=spring&logoColor=white" alt="Backend" /> | <img src="https://img.shields.io/badge/Backend-007ACC?style=for-the-badge&logo=spring&logoColor=white" alt="Backend" /> | <img src="https://img.shields.io/badge/Backend-007ACC?style=for-the-badge&logo=spring&logoColor=white" alt="Backend" /> | <img src="https://img.shields.io/badge/Backend-007ACC?style=for-the-badge&logo=spring&logoColor=white" alt="Backend" /> | <img src="https://img.shields.io/badge/Backend-007ACC?style=for-the-badge&logo=spring&logoColor=white" alt="Backend" /> |
+|                                                                                                                 사용자 관리 API 구현                                                                                                                 |                                                       대시보드 API 구현                                                       |                                                        리뷰 API 구현                                                        |                                                        도서 API 구현                                                        |                                                        댓글 API 구현                                                        |                                                        알림 API 구현                                                        |
+
+---
+
+## 기술 스택
+
+### 백엔드
+
+<!-- Spring Boot, Spring Data JPA -->
+<img src="https://img.shields.io/badge/Spring%20Boot-6DB33F?style=for-the-badge&logo=Spring%20Boot&logoColor=white" /> <img src="https://img.shields.io/badge/Spring%20Data%20JPA-6DB33F?style=for-the-badge&logo=Spring&logoColor=white" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)